lahja/asyncio/endpoint.py

```diff
@@ -612,10 +616,11 @@ def stop(self) -> None:
     ) -> None:
         item.bind(self, id)
 
+        await self._process_item(item, config)
```

Comment:
I think this is ok for this PR but we might want to reconsider this. The implication of this is that every broadcasted event (some are high frequency) is going into `_process_item` which doesn't do much if the endpoint isn't subscribed itself but it does do *something* 

I *think* we should probably only process it if we are actually subscribed to the event (similar handling as if it was an external event that we are subscribed for)